airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/poetry/commands.py

```diff
@@ -0,0 +1,102 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+"""
+Module exposing the format commands.
+"""
+from __future__ import annotations
+
+import logging
+import sys
+from typing import Any, Dict, List, Optional
+
+import asyncclick as click
+from pipelines.airbyte_ci.format.configuration import FORMATTERS_CONFIGURATIONS, Formatter
+from pipelines.airbyte_ci.format.format_command import FormatCommand
+from pipelines.cli.click_decorators import click_ignore_unused_kwargs, click_merge_args_into_context_obj
+from pipelines.cli.dagger_pipeline_command import DaggerPipelineCommand
+from pipelines.helpers.cli import LogOptions, invoke_commands_concurrently, invoke_commands_sequentially, log_command_results
+from pipelines.models.contexts.click_pipeline_context import ClickPipelineContext, pass_pipeline_context
+from pipelines.models.steps import StepStatus
+
+from .pipeline import PublishToPyPI, PyPIPublishContext
+
+
+@click.group(
+    name="poetry",
+    help="Commands related to running poetry commands.",
+)
+@click.option(
```

Comment:
> it might be beneficial to directly publish in some cases (especially in early phases).

In these cases would you publish from a local airbyte-ci execution?
If yes, I would suggest adding a user prompt instead of logging a warning.

Reply:
Yes, exactly. Makes sense to me, switching to a prompt